src/test/txpackage_tests.cpp

```diff
@@ -568,6 +587,205 @@ BOOST_FIXTURE_TEST_CASE(package_witness_swap_tests, TestChain100Setup)
         BOOST_CHECK(!m_node.mempool->exists(GenTxid::Wtxid(ptx_parent2_v1->GetWitnessHash())));
         BOOST_CHECK(m_node.mempool->exists(GenTxid::Txid(ptx_parent3->GetHash())));
         BOOST_CHECK(m_node.mempool->exists(GenTxid::Txid(ptx_mixed_child->GetHash())));
+
+        // package feerate should include parent3 and child. It should not include parent1 or parent2_v1.
+        BOOST_CHECK(mixed_result.m_package_feerate != std::nullopt);
+        CFeeRate expected_feerate(1 * COIN, GetVirtualTransactionSize(*ptx_parent3) + GetVirtualTransactionSize(*ptx_mixed_child));
+        BOOST_CHECK_MESSAGE(mixed_result.m_package_feerate.value() == expected_feerate,
+                            strprintf("Expected package feerate %s, got %s", expected_feerate.ToString(),
+                                      mixed_result.m_package_feerate.value().ToString()));
+    }
+}
+
+BOOST_FIXTURE_TEST_CASE(package_cpfp_tests, TestChain100Setup)
+{
+    mineBlocks(5);
+    LOCK(cs_main);
+    unsigned int expected_pool_size = m_node.mempool->size();
```

Comment:
22bf054
- prefer absolute references to cs_main in new code
- use `size_t` type for `size()` (or `auto`), same for line 216
```diff
-    LOCK(cs_main);
-    unsigned int expected_pool_size = m_node.mempool->size();
+    LOCK(::cs_main);
+    size_t expected_pool_size{m_node.mempool->size()};
```

Reply:
Done, noted